internal/pkg/cli/app_show.go

```diff
@@ -72,109 +77,121 @@ type ShowAppOpts struct {
 	if err != nil {
 		return fmt.Errorf("getting application: %w", err)
 	}
-	opts.app = serializedWebApp{
+	o.app = serializedWebApp{
 		AppName: app.Name,
 		Type:    app.Type,
-		Project: opts.ProjectName(),
-		Account: opts.proj.AccountID,
+		Project: o.ProjectName(),
 	}
 
-	environments, err := opts.storeSvc.ListEnvironments(opts.ProjectName())
+	environments, err := o.storeSvc.ListEnvironments(o.ProjectName())
 	if err != nil {
 		return fmt.Errorf("listing environments: %w", err)
 	}
 
-	var serializedEnvs []serializedWebAppEnv
-	var serializedServices []serializedWebAppService
+	var serializedWebAppRoutes []serializedWebAppRoute
+	var serializedDeployConfigs []serializedWebAppConfig
 	for _, env := range environments {
-		webAppURI, err := opts.describer.URI(env.Name)
+		webAppURI, err := o.describer.URI(env.Name)
 		if err == nil {
-			serializedEnvs = append(serializedEnvs, serializedWebAppEnv{
-				Name:   env.Name,
-				Region: env.Region,
-				Prod:   env.Prod,
-				URL:    webAppURI.DNSName,
-				Path:   webAppURI.Path,
+			serializedWebAppRoutes = append(serializedWebAppRoutes, serializedWebAppRoute{
+				Environment: env.Name,
+				URL:         webAppURI.DNSName,
+				Path:        webAppURI.Path,
 			})
-			webAppECSParams, err := opts.describer.ECSParams(env.Name)
+			webAppECSParams, err := o.describer.ECSParams(env.Name)
 			if err != nil {
 				return fmt.Errorf("retrieving application deployment configuration: %w", err)
 			}
-			serializedServices = append(serializedServices, serializedWebAppService{
-				Port:   webAppECSParams.ContainerPort,
-				Tasks:  webAppECSParams.TaskCount,
-				CPU:    webAppECSParams.CPU,
-				Memory: webAppECSParams.Memory,
+			serializedDeployConfigs = append(serializedDeployConfigs, serializedWebAppConfig{
+				Environment: env.Name,
+				Port:        webAppECSParams.ContainerPort,
+				Tasks:       webAppECSParams.TaskCount,
+				CPU:         webAppECSParams.CPU,
+				Memory:      webAppECSParams.Memory,
 			})
 			continue
 		}
 		if !applicationNotDeployed(err) {
 			return fmt.Errorf("retrieving application URI: %w", err)
 		}
 	}
-	opts.app.Environments = serializedEnvs
-	opts.app.Services = serializedServices
+	o.app.Configurations = serializedDeployConfigs
+	o.app.Routes = serializedWebAppRoutes
+
+	if o.shouldOutputInfra {
+		var serializedWebAppInfras []serializedWebAppInfra
+		for _, env := range environments {
+			webAppResources, err := o.describer.StackResources(env.Name)
+			if err == nil {
+				serializedWebAppInfras = append(serializedWebAppInfras, serializedWebAppInfra{
+					Environment:      env.Name,
+					ServiceID:        webAppResources.Service,
+					TargetGroupID:    webAppResources.TargetGroup,
+					TaskDefinitionID: webAppResources.TaskDefinition,
+				})
+				continue
+			}
+			if !applicationNotDeployed(err) {
+				return fmt.Errorf("retrieving application infrastructures: %w", err)
+			}
+		}
+		o.app.Infrastructures = serializedWebAppInfras
+	}
 
 	return nil
```

Comment:
nit: style, I prefer to return the struct at the end instead of initializing fields in multiple locations
```go
return &serializedWebApp{
	AppName: app.Name,
	...
	Resources: resources,
}, nil
```